tui/installer/screens.py

```diff
@@ -249,6 +254,16 @@ def get_installation_type(answers):
         entries.append(("Restore %s from backup to %s" % (b, diskutil.getHumanDiskLabel(b.root_disk)),
                         (b, None)))
 
+    # if we were not already told to enable RAID, and any of the disks
+    # found holds a MD signature, propose to activate
+    if "assemble-raid" not in answers:
+        for disk in diskutil.getQualifiedDiskList():
+            rv = util.runCmd2([ 'mdadm', '--examine', disk ])
+            if rv == 0:
```

Comment:
mdadm exit code is not enough, some disks don't have a md superblock but that command still returns 0:
```
INFO     [2022-12-15 15:31:50] ran ['mdadm', '--examine', '/dev/nvme0n1']; rc 0
STANDARD OUT:
/dev/nvme0n1:
   MBR Magic : aa55
Partition[0] :    104857599 sectors at            1 (type ee)

INFO     [2022-12-15 15:31:50] Citrix Hypervisor 8.2 CU1: disk /dev/nvme0n1 in ['/dev/nvme0n1'], skipping
INFO     [2022-12-15 15:31:50] Found a MD RAID on: /dev/nvme0n1
```